src/agent/agent.c

```diff
@@ -597,21 +597,23 @@ bool agent_parse_config(Agent *agent, const char *configfile) {
                 }
         }
 
+        return true;
+}
 
-        // set logging configuration
-        bc_log_init(agent->config);
-
+bool agent_set_config_values(Agent *agent) {
```

Comment:
Wouldn't `agent_apply_config_values` be a more descriptive name?

Reply:
Good point! Changed the function name.